packages/terra-popup/src/PopupArrow.module.scss

```
@@ -3,82 +3,82 @@
   [dir=rtl] {
     // These are relative to screen coordinates, so no bi-di concerns
     .popup-arrow {
-      border: var(--terra-popup-arrow-size, 11px) solid transparent;
+      border: var(--terra-popup-arrow-border, 11px) solid transparent;
```

Let's update this to:
```suggestion
      border-color: transparent;
      border-style: solid;
      border-width: var(--terra-popup-arrow-border-width, 11px); 
```